src/pages/templates/pages/partial/dashboard_panel_popular_reads_data.html

```diff
@@ -1,32 +1,34 @@
 <ul class="nav box-tabs mb-0">
-    <li class="nav-item"><button type="button" data-clusive-view="popular" class="nav-link popular-read-view-button{% if data.view == 'popular' %} active{% endif %}">Popular reads</button></li>
-    <li class="nav-item"><button type="button" data-clusive-view="assigned" class="nav-link popular-read-view-button{% if data.view == 'assigned' %} active{% endif %}">Assigned reads</button></li>
+    <li class="nav-item"><button type="button" data-clusive-view="assigned" class="nav-link popular-read-view-button{% if data.view == 'assigned' %} active{% endif %}">Assigned</button></li>
+    {% if not is_teacher %}
+    <li class="nav-item"><button type="button" data-clusive-view="recent" class="nav-link popular-read-view-button{% if data.view == 'recent' %} active{% endif %}">Recent</button></li>
+    {% endif %}
+    <li class="nav-item"><button type="button" data-clusive-view="popular" class="nav-link popular-read-view-button{% if data.view == 'popular' %} active{% endif %}">Trending</button></li>
```

Comment:
"Trending" should be "Popular" for teachers now as well, per design.

Reply:
Okay, done.